### PR TITLE
Add Quizet & G+, Update multiple sites

### PR DESCRIPTION
**Pull request by ChristoCoder**
I added quizlet and google+, and I also updated gmail, google, gopetition, change.org, cryptfolio, droplr, and audiomack

### CODE REVIEW

File: sites.json [hunk 1]
```diff
@@ -4199,7 +4206,14 @@
           "quip.com"
         ]
     },
-
+    {
+      "name": "Quizlet",
+      "url": "https://quizlet.com/delete-account",
+      "difficulty" : "easy",
+      "domains" : [
+ +            "quizlet.com"
```
Comment (tupaschoal) [inline sites.json]: Extra '+' and wrong indentation.

File: sites.json [hunk 2]
```diff
@@ -4199,7 +4206,14 @@
...
+ +    ]
```
Comment (tupaschoal) [inline sites.json]: Extra '+' and wrong indentation.

File: sites.json [hunk 3]
```diff
@@ -2128,25 +2129,31 @@
...
+        "url": "https://security.google.com/settings/security/deleteaccount",
         "difficulty": "easy",
         "domains": [
             "google.com",
             "google.co.uk"
         ]
     },
-
     {
-        "name": "GoPetition",
```
Comment (tupaschoal) [inline sites.json]: This cannot be deleted, it is missing from the GoPetition set below.

File: sites.json [hunk 4]
```diff
@@ -2128,25 +2129,31 @@
...
-        "url": "http://www.gopetition.com/",
+        "name" : "Google+",
+        "url" : "https://plus.google.com/downgrade",
+        "difficulty" : "easy",
+        "domains" : [
+            "plus.google.com"  
+        ]
+    },
+    {
+        "url": "https://www.gopetition.com/accountclosure.php",
```
Comment (tupaschoal) [inline sites.json]: The GoPetition name should be above this

File: sites.json [hunk 5]
```diff
@@ -2128,25 +2129,31 @@
...
-        "name": "GoPetition",
-        "url": "http://www.gopetition.com/",
+        "name" : "Google+",
+        "url" : "https://plus.google.com/downgrade",
+        "difficulty" : "easy",
+        "domains" : [
+            "plus.google.com"  
+        ]
+    },
+    {
```
Comment (tupaschoal) [inline sites.json]: Add a space between the end of one entry and beginning of another.

**Comment by tupaschoal** [comment]
Hello @ChristoCoder , are you willing to fix this?

**Comment by Nightfirecat** [comment]
Unless there's anything that needs to be verified, I'd be happy to open a PR with these changes committed under @ChristoCoder's name.

**Comment by tupaschoal** [comment]
Feel free to do so @Nightfirecat :)

**Comment by tupaschoal** [comment]
Was pushed with #78

**Comment by ChristoCoder** [comment]
Sorry about not responding, was having some issues with my mobile authenticator as I had just got a new phone

**Comment by tupaschoal** [comment]
It's alright @ChristoCoder , @Nightfirecat handled it with your login :)

**Pull request closed without merging**